sr_robot_launch/launch/sr_bimanual_ur10arms_hands.launch

```diff
@@ -55,6 +55,8 @@
   <!-- The control mode PWM (true) or torque (false) -->
   <!-- Set to true by default for now as torque control is not available yet -->
   <arg name="pwm_control" default="true"/>
+  <!-- Set the hand type here: hand_e_plus, hand_e, Hand_lite, hand_extra_lite -->
```

Comment:
we should make sure these examples are all lower-case, unless we plan on somehow adding support for mixed-case strings